src/index.html

```diff
@@ -10,8 +10,188 @@
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post"
+    class="form">
+      <fieldset class="form__group">
+        <legend>Personal information</legend>
+
+        <label class="form__field">
+          Surname:
+
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form__field">
+          Name:
+
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+            required
+          >
+        </label>
+
+        <label class="form__field">
+          How old are You?
+
+          <input
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+
+        <label class="form__field">
+          Full date of birth:
+
+          <input
+            type="date"
+            name="birthdate"
+          >
+        </label>
+
+        <div>
+          <label>
+            I accept the term of the agreement
+
+            <input
+              type="checkbox"
+              name="agreement"
+              required
+            >
+          </label>
+        </div>
```

Comment:
Why you wrapped this label with div element instead of adding form_field class as for other labels? You should try to be consistant with the way you write your code :)